pkg/metastore/badger.go

```diff
@@ -481,6 +485,39 @@ func (m *BadgerMetastore) GetOrCreateStacktraces(ctx context.Context, r *pb.GetO
 		stacktraceKeys = append(stacktraceKeys, MakeStacktraceKey(stacktrace))
 	}
 
+	const maxRetries = 2
```

Comment:
not sure about this setting

Reply:
I think this is ok, if people frequently run into problems we can make this a flag ultimately, but the less configuration is available the better.